src/group_theory/group_action/defs.lean

```diff
@@ -75,11 +90,11 @@ library_note "bundled maps over different rings"
 
 /-- Commutativity of actions is a symmetric relation. This lemma can't be an instance because this
 would cause a loop in the instance search graph. -/
-lemma smul_comm_class.symm (M N α : Type*) [has_scalar M α] [has_scalar N α]
+@[to_additive] lemma smul_comm_class.symm (M N α : Type*) [has_scalar M α] [has_scalar N α]
```

Comment:
Should the additive version have a docstring?

Reply:
Done (see below).